### DISCUSSION TITLE
Giudizio sospeso

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#742] [Richieste di aiuto]
Vorrei delle informazioni sugli scrutini per i giudizi sospesi. Come funzionano e quando sarà possibile svolgerli.

**Comment by registro-giuaschool** [comment]
Salve, devi accedere da amministratore, su **SCUOLA > SCRUTINI** e devi cliccare _Esami del giudizio sospeso__ e devi inserire la data dell'inizio dello scrutinio e la data per l'inserimento della proposta di voto.
I docenti delle materie con debito dovranno inserire le proposte di voto da **LEZIONI > SCRUTINIO** .
Quando tutte le proposte sono state inserite, il coordinatore o un membro dello staff apre lo scrutinio.
In caso di altri problemi sono disponibile alla seguente e-mail: registroelettronicogiuaschool@gmail.com

**Comment by luigiadoc** [comment]
Ho inserito la data di oggi come inizio per le proposte di voto ma, andando su LEZIONI > SCRUTINIO, non risultano inseribili.
@trinko Mi sembra di aver letto che uscirà una nuova versione per questo, giusto ?

**Reply by registro-giuaschool** [reply]
Può allegarmi uno screenshot delle date corrette per Esame giudizio sospeso e la pagina LEZIONI > SCRUTINIO del docente che ha inserito il debito?

**Reply by registro-giuaschool** [reply]
Contattami alla mia email: registroelettronicogiuaschool@gmail.com in modo da aiutarti più facilmente.

**Reply by trinko** [reply]
> Ho inserito la data di oggi come inizio per le proposte di voto ma, andando su LEZIONI > SCRUTINIO, non risultano inseribili. @trinko Mi sembra di aver letto che uscirà una nuova versione per questo, giusto ?

Per svolgere lo scrutinio, se non puoi aspettare la nuova versione, devi fare tutti gli aggiornamenti già presenti e successivi alla 1.5.8
